swatch-product-configuration/src/main/resources/subscription_configs/Ansible/ansible-aap-managed.yaml

```diff
@@ -26,3 +26,4 @@ metrics:
   - id: Instance-hours
     type: counter
     awsDimension: infrastructure
+    enableGratisUsage: true
```

Comment:
This should be set on `Managed-nodes` rather than `Instance-hours`

Reply:
Interesting! Agreed that's what I thought I had it before when I merged the main branch to this. :)

Reply:
Thanks for spotting! Yes, this was messed up after rebasing to merge :(